src/servers/src/http/otlp.rs

```diff
@@ -34,13 +43,18 @@ use super::header::{write_cost_header_map, CONTENT_TYPE_PROTOBUF};
 use crate::error::{self, Result};
 use crate::query_handler::OpenTelemetryProtocolHandlerRef;
 
+const OTLP_GREPTIME_LOG_PIPELINE_NAME_HEADER_NAME: &str = "x-greptime-log-pipeline-name";
+const OTLP_GREPTIME_LOG_PIPELINE_VERSION_HEADER_NAME: &str = "x-greptime-log-pipeline-version";
+const OTLP_GREPTIME_LOG_TABLE_NAME_HEADER_NAME: &str = "x-greptime-log-table-name";
```

Comment:
Suggesting move these to `src/servers/src/http/header.rs` and removing the `OTLP` prefix. We may reuse them in other scenarios like Loki log write ingestion.